coupling/indexing/IndexingService.h

```diff
@@ -160,4 +164,5 @@ template <unsigned int dim> class coupling::indexing::IndexingService {
 #if (COUPLING_MD_ERROR == COUPLING_MD_YES)
   bool _isInitialized = false;
 #endif
+  friend IndexingServiceTest;
```

Comment:
I just read today that this was really dirty to do that :lollipop:

Reply:
Yes it is a bit hacky. The proper way to do this is to move it (getUniqueRankForMacroscopicCell) to another class, e.g. `CellRankMap` or so. Then CellRankMap can have unit tests, and the IndexingService can have a private member of type CellRankMap and call its functions. Alternatively, perhaps the content of getUniqueRankForMacroscopicCell() can perhaps be integrated into getRanksForGlobalIndex() (?). However, since this is a bit of refactoring work, and it works fine for now, I have created an issue (#28 ) for it